doc/level1.rst

```diff
@@ -297,10 +297,21 @@ concerns, the user id SHOULD NOT be an empty string.
 These packets MUST be assembled in binary format (not ASCII-armored),
 and then base64-encoded.
 
-A Level 1 MUA MUST be capable of processing and handling 2048-bit and
-3072-bit RSA public keys.  It MAY support other OpenPGP key formats
-found in an Autocrypt header (for example, by passing it agnostically
-to an OpenPGP backend for handling).
+A Level 1 MUA MUST be capable of processing and handling Ed25519
```

Comment:
shouldn't this read "A Level 1.1 MUA ..."?

Reply:
Nope. This is still level 1, but version 1.1 of it.

Reply:
yeah, as mentioned, i understand. but still find this confusing ;)
but it's fine for me :)